5.PropTypes and DefaultProps.md

```diff
@@ -0,0 +1,40 @@
+#Prop Types And Default Props
+
+##Prop Types
+PropTypes provides **typechecking ability** to react components.
+Let's take a look at some codes.
+
+```
+class Jam3Lesson extends React.Components{
+  render(){
+    return (
+      <div>
+        <div>{this.props.title}</div>
+        <div>{this.props.desc}</div>
+        ...
+      </div>
+    )
+  }
+}
+
+// The propTypes typechecking happens after defaultProps are resolved,
+// so typechecking will also apply to the defaultProps.
+Jam3Lesson.propTypes = {
+  title: propTypes.string.isRequired //Required prop
+  desc: propTypes.string             //optional prop
+  ...
+}
+```
+
+##Default Props
+With defaultProps, you can provide default value for the props so that a component works well without receiving props.
+
+```
+Jam3Lesson.defaultProps = {
+  desc: 'Please provide description of this lesson'
+}
+
+```
+
+[List of PropTypes](https://facebook.github.io/react/docs/typechecking-with-proptypes.html#react.proptypes)<br />
+[Default Props](https://facebook.github.io/react/docs/typechecking-with-proptypes.html#default-prop-values)
```

Comment:
Leave a line break at the end, and ad references at the end of the article with a h3 heading saying `References` as an unordered list.